ibis/file/hdf/tests/test_basic.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import pandas as pd
+import ibis
+
+from pandas.util import testing as tm
+pytest.importorskip('tables')
+
+from ibis.file.hdf.client import HDFClient, HDFTable  # noqa: E402
+from ibis.file.client import Database, execute  # noqa: E402
+
+
+@pytest.fixture
+def transformed(hdf):
+
+    closes = hdf.hdf.prices.close
```

Comment:
Can you call the second `hdf` in `hdf.hdf` something else?

Reply:
done